generator/explores/__init__.py

```diff
@@ -1,9 +1,17 @@
 """All possible explore types."""
 from .explore import Explore  # noqa: F401
+from .glean_ping_explore import GleanPingExplore
 from .growth_accounting_explore import GrowthAccountingExplore
 from .ping_explore import PingExplore
 
 EXPLORE_TYPES = {
+    GleanPingExplore.type: GleanPingExplore,
     PingExplore.type: PingExplore,
     GrowthAccountingExplore.type: GrowthAccountingExplore,
 }
+
+GLEAN_EXPLORE_TYPES = {
+    t: v
+    for (t, v) in EXPLORE_TYPES.items()
+    if t in [GleanPingExplore.type, GrowthAccountingExplore.type]
+}
```

Comment:
I don't think we have to go the path of explicitly separating these anymore - I added `glean_app: $bool` to the `namespaces.yaml` entries, we can check in the namespace and only return an instance of the view/explore if it is a Glean application.

Reply:
Yeah, this was a relic of my work on Friday that I forgot to clean up. I'll take it out.

Reply:
Actually I think `GLEAN_VIEW_TYPES` might be the most elegant way of making sure we don't generate duplicate views for the same Glean ping. I did take out `GLEAN_EXPLORE_TYPES` though.

Reply:
Specifically: https://github.com/mozilla/lookml-generator/pull/58/files#diff-a6989594893a2eff2aa73d1affe2a11cc1a85effed9ef992a025847501788140R107

Reply:
Mmm, Let's discuss more later, but in general I prefer the logic of  "Does this definition generate a view for this table" to live in one place.